File: java/Easy/CountOddNumbersInAnInterval.java
package Easy;

public class CountOddNumbersInAnInterval {

    public static void main(String[] args) {
        System.out.println(countOdds(0, 9));
    }
    public static int countOdds(int low, int high){
        return (high + 1)/2 - low/2;
    }
}

File: kotlin/Easy/CountOddNumbersInAnInterval.kt
package Easy

class CountOddNumbersInAnInterval{
    fun countOdds(low: Int, high: Int): Int {
        return (high + 1)/2 - low/2
    }
}

fun main () {
    var a = CountOddNumbersInAnInterval()
    println(a.countOdds(3, 5))
}